modin/core/io/text/json_glob_dispatcher.py

```diff
@@ -0,0 +1,99 @@
+# Licensed to Modin Development Team under one or more contributor license agreements.
+# See the NOTICE file distributed with this work for additional information regarding
+# copyright ownership.  The Modin Development Team licenses this file to you under the
+# Apache License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+"""Module houses ``JsonGlobDispatcher`` class that is used for reading `.json` files."""
```

Comment:
why is this not a fully "experimental" feature? it's accessible from Experimental factory but the file is not under `experimental` dir

Reply:
Good question!
I don't know. In the same way `read_csv_glob` was implemented.

Make sense to me transfer all experimental dispatchers and parsers under `experimental` dir.

Reply:
cc @YarShev

Reply:
Truthfully we need a way to distinguish between experimental and pandas extensions.